x-pack/plugins/observability/public/application/index.tsx

```diff
@@ -70,7 +75,9 @@ export const renderApp = (
           <EuiThemeProvider darkMode={isDarkMode}>
             <i18nCore.Context>
               <RedirectAppLinks application={core.application}>
-                <App />
+                <ObsvSharedContextProvider>
```

Comment:
Can we find a better name? _What_ shared data is this? Right now I can only assume it's a kitchen sink of mixed data.

Reply:
ok let me think about it.

Reply:
I created a new context called  `HasDataContextProvider`, is it clear @sqren ?